packages/ag-charts-community/src/dom/theme.css

```diff
@@ -18,11 +18,11 @@
     --ag-charts-border-critical: var(--ag-borders-critical, solid 1px);
 
     /* Layout */
-    --ag-charts-layer-ui-overlay: 5;
-    --ag-charts-layer-tooltip: 4;
-    --ag-charts-layer-toolbar: 3;
-    --ag-charts-layer-crosshair: 2;
-    --ag-charts-layer-annotations: 1;
+    --ag-charts-layer-ui-overlay: 50;
```

Comment:
What's the reason for the increase?

Reply:
I needed to nest a color picker within a dialog popover, so it adds 1 to the layer when nesting. Wanted to avoid conflicts with other layers.